### PR TITLE
Nerfs gaseous phoron across the station as well as some smaller tweaks

### PR DESCRIPTION
**Pull request by Hockaa**
Given the whole phoron scarcity going on I felt the need to nerf the amount of gaseous phoron the station has since we have ridiculous amounts of it.

The amount of gas that starts in atmospheric;'s phoron reserve tank has been drastically reduced - by my own estimate you can squeeze out about 10-12 canisters (at about 4600 kPa) before running out - due to the lowered amount the gas systems will also take longer to fill canisters up.

A new childtype of phoron canisters has been introduced which spawn with half the amount regular phoron canisters do. While most of the roundstart station canisters have been replaced with totally empty canisters these will still spawn so the departments are not entirely devoid of phoron. The exact numbers are: 3 'scarce' canisters in engineering (1 in hard storage, 2 in atmos sublevel) - 2 in science (1 in anomaly lab, 1 in toxins)

There are two somewhat miscellaneous changes this PR comes with:
The four nitrogen canisters that used to start in the supermatter engine room have been reduced down to two canisters and now have windows, windoors marked as 'Emergency Coolant' and warning tape around them to more clearly define they are for emergency use, hopefully meaning less fledgling apprentices try using it in the supermatter as routine coolant.

The childtype of canisters that spawned empty (and were marked as certain gas types already) weren't correctly set to update their icons, so they would spawn without having the light overlay. I solved this by removing all instances of canister childs using src.update_icon and instead making canister initialize() do a late initialize, and updating the icon there.

### CODE REVIEW

**Comment by Hockaa** [comment]
!review

File: html/changelogs/hockaa-PHnerf.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+author: Hocka
+
+delete-after: True
+
+changes:
+  - maptweak: "All roundstart phoron canisters were removed from the station. There are now three empty phoron canisters in engineering hard storage with one half-capacity canister. Toxins and the Anomaly lab have received a similar treatment."
+  - maptweak: "The reserve phoron tank in atmos has had it's starting amount significantly reduced."
+  - maptweak: "The four Nitrogen canisters in the engine room have been narrowed down to just 2 canisters and are now more clearly marked as emergency use."
+  - bugfix: "Default empty canisters now start with their lights correctly."
```
Comment (Doxxmedearly) [inline html/changelogs/hockaa-PHnerf.yml:9]: correctly what? Don't leave me in suspense!

**Review by Doxxmedearly — APPROVED** [review]

**Review by Chada-1 — APPROVED** [review]

File: code/game/machinery/atmoalter/canister.dm [hunk 2]
```diff
@@ -51,6 +51,8 @@
 	canister_color = "blue"
 	can_label = 0
 
+/obj/machinery/portable_atmospherics/canister/phoron/scarce
```
Comment (NonQueueingMatt) [inline code/game/machinery/atmoalter/canister.dm:54]: why are there 2 empty paths of this?

**Review by NonQueueingMatt — DISMISSED** [review]

File: code/game/machinery/atmoalter/canister.dm (hunk 2 above)
Comment (Hockaa) [reply code/game/machinery/atmoalter/canister.dm]: removed in the next commit

**Pull request merged**